Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet/App.xaml.cs

```diff
@@ -74,6 +74,12 @@ protected override void OnStart()
             MobileCenter.Start($"uwp={uwpKey};android={androidKey};ios={iosKey}",
                                typeof(Analytics), typeof(Crashes), typeof(Distribute), typeof(Push));
 
+            CustomProperties properties = new CustomProperties();
+            properties.Set("a", "b").Clear("a").Set("decimal", 3.000001M);
```

Comment:
Can you add invalid cases in the test? For instance, key not matching regex, empty/null key, etc.